### DISCUSSION TITLE
KIPET using Ipopt

### DISCUSSION THREAD

**Discussion by timhockswender** [coin-or/Ipopt#525] [Q&A] [answered]
I am trying to run kipet under windows 10.  Installed Ipopt via conda. kipet example 1 runs fine. Example 2 fails, seems to be looking for libhsl.dll.  Its not in my system. Not sure where to get it or put.
Advice appreciated.
Tim

**Comment by svigerske** [comment, marked as answer]
This usually comes up if one has selected a HSL linear solver that hasn't been build into Ipopt. Ipopt then looks for a library where it can load this from at runtime: https://coin-or.github.io/Ipopt/INSTALL.html#LINEARSOLVERLOADER

Either provide such a library or select a different linear solver (https://coin-or.github.io/Ipopt/OPTIONS.html#OPT_linear_solver).

**Reply by timhockswender** [reply]
Thanks for getting back to me. I am not requesting any other solver. I am just trying to run example 2. Can you suggest what to do in this context?
Tim

**Reply by svigerske** [reply]
No. I don't know anything about kipet or about the Ipopt packages in conda. This is the discussion forum of Ipopt here. You might want to try with the kipet developers and check how they use Ipopt.

**Reply by timhockswender** [reply]
Ok.  Thanks.
